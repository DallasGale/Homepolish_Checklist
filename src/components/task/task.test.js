import React from 'react';
import Task from './task.component';
import renderer from 'react-test-renderer'

describe('<Task />', () => {
  
    it('will render without exploading!', () => {
        const tree = shallow.create(<Task />).toJSON();
        expect(tree).toMatchSnapshot();
    });

})