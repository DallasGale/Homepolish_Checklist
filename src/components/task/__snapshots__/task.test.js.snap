// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Task /> will render without exploading! 1`] = `
<div
  className="sc-bxivhb gyNEet"
>
  <button
    className="sc-bdVaJa efOOzp"
    onClick={undefined}
    value={undefined}
  >
    <div
      className="sc-bwzfXH fvXDKP"
    >
      <div
        className="sc-htpNat ehtoVH"
      />
    </div>
  </button>
  <div
    className="sc-ifAKCX bSWLRl"
  />
</div>
`;
